frontend/src/components/EmailSubscriptionContainer.tsx

```diff
@@ -29,7 +29,7 @@ const EmailSubscriptionContainer: React.FC<EmailSubscriptionContainerProps> = ({
     return (
         <form onSubmit={handleSubmit}>
             <div className="email-section">
-                <p className="interest-text">If you are interested in following the community, please leave your email!</p>
+                <p className="interest-text">Se sei interessato a seguire la comunità, lascia la tua email!</p>
```

Comment:
Community is a bit misleading here, I would change it to
````
Se sei interessato a ricevere aggiornamenti, lascia la tua email!
````

Reply:
Fixed